### DISCUSSION TITLE
open shell settings

### DISCUSSION THREAD

**Discussion by rajones2** [Open-Shell/Open-Shell-Menu#2003] [Q&A]
I am an 82-year-old computer illiterate trying to follow instructions to put open windows on my taskbar.  I am lost. How do I "make sure that the "show open windows on the taskbar" is enabled?  Thank you in advance.

**Comment by JohnC654** [comment]
Not sure what you mean by "open windows". Are you talking about Open-Shell? If you want to use Open-Shell to modify your Start Menu, then 1. install Open-Shell, 2. Right-click on the Start Menu button to see the Open-Shell Menu and select "Settings". Once the Settings for Open-Shell window opens, Click on the "Start Menu Style" tab and choose the style you prefer (I prefer the Classic style, personally.) Then start going through each tab and make your decisions for all the options listed in each tab. There are a lot of things you can do there and it a good idea when you're done to click on the "Backup" button in the lower right-hand corner by the "OK" and "Cancel" buttons, then select "Save to XML File..." to back up your settings that you just chose. Hope this helps. And note that there are two parts to Open-Shell: 1. first there is Open-Shell Menu, which is a Start Menu "skin", and: 2. Settings for Classic Explorer, which is a skin for Windows File Explorer.
